recipes/datalad/meta.yaml

```diff
@@ -53,6 +54,7 @@ test:
     - datalad --help
     - git-annex-remote-datalad-archives --help
     - git-annex-remote-datalad --help
+    - python -m nose -s -v datalad
```

Comment:
This may require nose to be installed and the files to be local?

Reply:
nose is listed in `test.requires`, tests are spread out through `datalad` package under `tests/` submodules, so should be all ok I guess